dre-service/src/main/java/io/cdap/re/RulesDB.java

```diff
@@ -556,7 +557,7 @@ String generateRulebook(NamespacedId namespacedId)
     return String.format(RULEBOOK_TEMPLATE, rulebookRow.getString(ID_COL), rulebookRow.getLong(VERSION_COL),
                          rulebookRow.getString(DESCRIPTION_COL), rulebookRow.getLong(CREATED_COL),
                          rulebookRow.getLong(UPDATED_COL), rulebookRow.getString(SOURCE_COL),
-                         rulebookRow.getString(USER_COL), ruleOutput);
+                         rulebookRow.getString(USER_COL), Joiner.on("").join(ruleOutput));
```

Comment:
can do this in plain java:
```
ruleOutput.stream().collect(Collectors.joining(""))
```

Reply:
Since we decided to use Joiner again, should I leave the Joiner? or use the above statements?

Reply:
use the above

Reply:
IntelliJ is recommending to use `String.join("", ruleOutput)`. Should I use this suggestion instead?